apps/ejabberd/include/adhoc.hrl

```diff
@@ -33,10 +33,10 @@
                          node :: ejabberd:server(),
                          sessionid :: binary(),
                          status :: atom(),
-                         defaultaction = "" :: string(),
+                         defaultaction = <<>> :: binary(),
```

Comment:
By the way, can you rename it to `dafault_action` ? Probably, the same story couple lines above with `sessionid`